File: shape-image-view/src/main/java/com/wkz/shapeimageview/progress/OnProgressListener.java
package com.wkz.shapeimageview.progress;

import com.bumptech.glide.load.engine.GlideException;

/**
 * @author Administrator
 */
public interface OnProgressListener {

    void onProgress(String imageUrl, long bytesRead, long totalBytes, boolean isDone, GlideException exception);
}

File: shape-image-view/src/main/java/com/fphoenixcorneae/shapeimageview/progress/OnProgressListener.kt
package com.fphoenixcorneae.shapeimageview.progress

import com.bumptech.glide.load.engine.GlideException

interface OnProgressListener {
    fun onProgress(
        imageUrl: String?,
        bytesRead: Long,
        totalBytes: Long,
        isDone: Boolean,
        exception: GlideException?
    )
}